pkgs/development/python-modules/pysam/default.nix

```diff
@@ -8,35 +8,36 @@
 , htslib
 , libdeflate
 , xz
-, pytest
+, pytestCheckHook
 , samtools
 , zlib
 }:
 
 buildPythonPackage rec {
   pname   = "pysam";
-  version = "0.20.0";
+  version = "0.21.0";
 
   # Fetching from GitHub instead of PyPi cause the 0.13 src release on PyPi is
   # missing some files which cause test failures.
   # Tracked at: https://github.com/pysam-developers/pysam/issues/616
   src = fetchFromGitHub {
     owner = "pysam-developers";
-    repo = "pysam";
-    rev = "refs/tags/v${version}";
-    hash = "sha256-7yEZJ+iIw4qOxsanlKQlqt1bfi8MvyYjGJWiVDmXBrc=";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-C4/AJwcUyLoUEUEnsATLHJb5F8mltP8X2XfktYu0OTo=";
   };
 
   nativeBuildInputs = [ samtools ];
   buildInputs = [
     bzip2
     curl
-    cython
     libdeflate
     xz
     zlib
   ];
 
+  propagatedBuildInputs = [ cython ];
```

Comment:
Looks like that `cython` should go into `nativeBuildInputs` as it's not a runtime requirement.

Reply:
Moved.

Reply:
I agree that it is not runtime dependencies, but it seems that the libraries that depend on "pysam" (deeptools and truvari) require cython to be in the propagatedBuildInputs.